app/controllers/api/v1/release_engines/rubygems/specs_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Api::V1::ReleaseEngines
+  class Rubygems::SpecsController < Api::V1::BaseController
+    before_action :scope_to_current_account!
+    before_action :require_active_subscription!
+    before_action :authenticate_with_token
+    before_action :set_artifact
+
+    def quick
+      authorize! artifact,
+        to: :show?
+
+      # rubygems expects a marshalled and gzipped gemspec
+      gemspec = artifact.specification.as_gemspec
+      dump    = Marshal.dump(gemspec)
+      gz      = Zlib::Deflate.deflate(dump)
```

Comment:
`gz` is 'incorrect', maybe use `rz` or `compressed` — or create a `send_compressed_data` method?